daemon/cmd/daemon_main.go

```diff
@@ -1659,6 +1666,33 @@ func initKubeProxyReplacementOptions() {
 			option.Config.NodePortMode = option.NodePortModeSNAT
 		}
 	}
+
+	if option.Config.EnableSessionAffinity {
+		if !probesManager.GetMapTypes().HaveLruHashMapType {
+			msg := "SessionAffinity feature requires BPF LRU maps"
+			if strict {
+				log.Fatal(msg)
+			} else {
+				log.Warn(fmt.Sprintf("%s. Disabling the feature.", msg))
```

Comment:
`log.Warnf("%s. Disabling the feature.", msg)`

Reply:
Will do it in a follow-up.